changelog/11118.improvement.rst

```diff
@@ -0,0 +1,3 @@
+Allow plugins to be loaded with `-p` from paths specified in `pythonpath`.
```

Comment:
Now :confval:`pythonpath` configures `$PYTHONPATH` earlier than before during the initialization process, which now also affects plugins loaded via the `-p` command-line option.

Reply:
(That was meant as a "suggestion" before)